bin/dump_wiki

```diff
@@ -38,9 +41,72 @@ if (argv.h) {
 // Strip a trailing slash
 argv.host = argv.host.replace(/\/$/, '');
 
-argv.ns = Number(argv.ns);
+// Default global options
+var options = { maxConcurrency : 50, concurrency: 3 };
+
+if (argv.c) {
+    // Get config, or throw exception on error
+    try {
+        var conf = yaml.safeLoad(fs.readFileSync(argv.c));
+        Object.keys(conf.dump_wiki).forEach(function(arg){
+            if (!argv[arg]) {
+                argv[arg] = conf.dump_wiki[arg];
+            }
+        });
+        options = conf.global_opts;
+    } catch (e) {
+      console.log(e);
+    }
+
+    // Check if required params are missing
+    if (!argv.apiURL || !argv.domain || !argv.ns || !argv.host) {
+        console.log('[Error - One of the required params is missing in config]');
+        process.exit(1);
+    }
+
+    // Set defaults
+    if (!argv.d) {
+        argv.d = '';
+    }
+    if (!argv.db) {
+        argv.db = '';
+    }
+    if (!argv.verbose) {
+        argv.verbose = true;
+    }
+
+} else {
+    //check whats missing and show help
+    argParser.reset()
+        .usage('Create a HTML dump in a subdir\n'
```

Comment:
C/p is not the proper solution here. Collect all of the given command-line args, parse the config if given, and then check if the script has all of the needed info to continue.